en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md

```diff
@@ -13,23 +13,19 @@ The Gateway of an API Manager deployment is responsible for the main business fu
 
 The handlers are request and response interceptors. The list of API handlers in WSO2 API-M are as follows:
 
--   APIMgtLatencyStatsHandler
 -   CORSRequestHandler
 -   APIAuthenticationHandler
 -   ThrottleHandler
--   APIMgtUsageHandler
```

Comment:
Let's add the new handler name here

Reply:
Will send a new pr with the new handler info.